### PR TITLE
Added the header file + modified main.cpp

### PR DESCRIPTION
**Pull request by JackLi64842**
testing

### CODE REVIEW

File: headers/yli.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+void spaces_in_file() {
+  int num_spaces = 0;
+  char character;
+  string filename;
+  ifstream file;
+
+  cout << "Enter the name of a file(Example: hello.txt): ";
+  cin >> filename;
+  file.open(filename);
+  if (filename) {
```
Comment (KrumphauChicken) [inline headers/yli.h]: Strings cannot be used as bools. This is not a file handle.

File: headers/yli.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    while (file >> character) {
+      if (character = " ") {
```
Comment (KrumphauChicken) [inline headers/yli.h]: Cannot compare character variable with a string literal.

Comment (KrumphauChicken) [inline headers/yli.h]: Also, this is an assignment, and you counted everything but spaces by using cin >> instead a .get().

File: main.cpp [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-int main()
-{
-	short unsigned int choice;
-	cout << "0) Convert examples to kilo-examples.\
+int main() {
+  short unsigned int choice;
+  cout << "0) Convert examples to kilo-examples.\
 	\n1) \
 	\n2) \
 	\n3) \
```
Comment (KrumphauChicken) [inline main.cpp]: Label the number.

Comment (KrumphauChicken) [inline main.cpp]: Label the number.

**Comment by JackLi64842** [comment]
Should I have not used '#3' as part of my 3rd commit?

**Comment by KrumphauChicken** [comment]
Those are merge conflicts, which I didn't cover in class. It means changes on the master branch conflict with changes on your branch. I fixed them, but someone merge to master just now while I was typing, and now there are more. The >>>>HEAD section shows your branch, and the >>>>master section shows the same lines how they are on the master branch. You have to edit the file by hand to reconcile the conflicts. These conflict will be easy because it's just everyone adding our includes on the same line; so just remove the >>>> and ==== lines.

**Pull request merged**